build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -230,10 +230,10 @@ CONFIG_BOARD_ASSERT_SYSTEM_HALT=y
 # SPI Flash driver
 #
 CONFIG_AMEBAD_FLASH_BASE=0x6000
-CONFIG_AMEBAD_FLASH_CAPACITY=0x27A000
+CONFIG_AMEBAD_FLASH_CAPACITY=0x28A000
```

Comment:
yes, we can set max flash capacity always, as suggested by Mr.Chang

Reply:
I agree lets keep maximum flash capacity to avoid future issues.